src/main/webapp/scripts/suggest-script.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Gets executed when button is clicked
+ */
+window.onload = function() {
+  document.getElementById('make-suggestion').onclick = function() {
+    makeSuggestion();
+  };
```

Comment:
nitpick: Since `onclick` takes in a function with zero arguments, and `makeSuggestion` is a function with zero arguments, you could simplify this by just referencing the named function directly:

`document.getElementById('make-suggestion').onclick = makeSuggestion;`

Reply:
Made change suggested.